Plots/Overlays/NCL_overlay_11a.py

```diff
@@ -0,0 +1,213 @@
+# -------------------------
+#
+# This allows Cartopy to transform ``lat`` and ``lon`` values accurately into
+# points on the matplotlib plot canvas.
+
+crs = PlateCarree()
+
+###############################################################################
+# Construct shape boundaries
+# --------------------------
+#
+# Using Cartopy's interface to the Natural Earth Collection of shapefiles
+# and geographical shape data, we construct the geographical boundaries
+# that we are interested in displaying, namely the country borders of China
+# and Taiwan, the borders of Chinese provinces, and all land borders *without*
+# China or Taiwan.
+
+# Download the Natural Earth shapefile for country boundaries at 10m resolution
+shapefile = natural_earth(category='cultural',
+                          resolution='10m',
+                          name='admin_0_countries')
+
+# Sort the geometries in the shapefile into Chinese/Taiwanese or other
+country_geos = []
+other_land_geos = []
+for record in ShapeReader(shapefile).records():
+    if record.attributes['ADMIN'] in ['China', 'Taiwan']:
+        country_geos.append(record.geometry)
+    else:
+        other_land_geos.append(record.geometry)
+
+# Define a Cartopy Feature for the country borders and the land mask (i.e.,
+# all other land) from the shapefile geometries, so they can be easily plotted
+countries = ShapelyFeature(country_geos,
+                           crs=crs,
+                           facecolor='none',
+                           edgecolor='black',
+                           lw=1.5)
+land_mask = ShapelyFeature(other_land_geos,
+                           crs=crs,
+                           facecolor='white',
+                           edgecolor='none')
+
+# Download the Natural Earth shapefile for the states/provinces at 10m resolution
+shapefile = natural_earth(category='cultural',
+                          resolution='10m',
+                          name='admin_1_states_provinces')
+
+# Extract the Chinese province borders
+province_geos = [record.geometry for record in ShapeReader(shapefile).records()
+                 if record.attributes['admin'] == 'China']
+
+# Define a Cartopy Feature for the province borders, so they can be easily plotted
+provinces = ShapelyFeature(province_geos,
+                           crs=crs,
+                           facecolor='none',
+                           edgecolor='black',
+                           lw=0.25)
+
+###############################################################################
+# Plot
+# ----
+
+# Create the figure and the Cartopy GeoAxes object
+fig = plt.figure(figsize=(12,12))
+ax = plt.axes(projection=crs)
+
+# Define the axis tick parameters and labels
+lon_formatter = LongitudeFormatter(zero_direction_label=False,
+                                   dateline_direction_label=False)
+lat_formatter = LatitudeFormatter()
+ax.xaxis.set_major_formatter(lon_formatter)
+ax.yaxis.set_major_formatter(lat_formatter)
```

Comment:
```python

gcv.util.add_lat_lon_ticklabels(ax)
```

(defaults to setting both `direction_label` variables to `False`)

Reply:
Done.